benchmark/benchmarkExactVertexColoring.cpp

```diff
@@ -0,0 +1,62 @@
+#include <cassert>
+#include <iostream>
+#include <map>
+
+#include <coloring/EnumerationVertexColoring.hpp>
```

Comment:
A general question for discussion: shouldn't it be ExactVertexColoring.hpp with EnumerationVertexColoring.cpp and (possibly) InclusionExclusionVertexColoring.cpp in the future?

Reply:
Seems like a good idea